source/features/quick-review.tsx

```diff
@@ -11,6 +11,7 @@ import observe from '../helpers/selector-observer.js';
 import showToast from '../github-helpers/toast.js';
 import {getConversationNumber, getUsername} from '../github-helpers/index.js';
 import {randomArrayItem} from '../helpers/math.js';
+import {getToken} from '../github-helpers/github-token.js';
```

Comment:
Does the refined-github requires any import-order convention?

Reply:
Not anymore 👌